src/Model/Mail.php

```diff
@@ -76,6 +76,10 @@ public static function insert($msg, $notifiction = true)
 
 		if ($msg['reply']) {
 			$reply = DBA::selectFirst('mail', ['uri', 'uri-id'], ['parent-uri' => $msg['parent-uri'], 'reply' => false]);
+			if (!DBA::isResult($reply)) {
```

Comment:
I would prefer a workaround where the mail is kept. Mails are really important, so we should try to keep them.

Reply:
I'll use the default case then.